Build/package-lock.json

```diff
@@ -2212,6 +2212,12 @@
       "resolved": "https://registry.npmjs.org/arr-union/-/arr-union-3.1.0.tgz",
       "integrity": "sha1-45sJrqne+Gao8gbiiK9jkZuuOcQ="
     },
+    "array-differ": {
```

Comment:
Did you mean to update this? The changes look unrelated.

Reply:
I did not update this manually, probably the build command did while I was testing.

Reply:
@soninaren I assume this should be reverted? It seems like something that should be updated in another PR if needed